modules/services/monitoring/netdata.nix

```
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.netdata;
+
+in {
+  meta.maintainers = [ "rsrohitsingh682" ];
+
+  options = {
+    services.netdata = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable Netdata Daemon.";
+      };
+
+      package = mkOption {
+        type = types.path;
+        default = pkgs.netdata;
+        description = "The package to use for Netdata";
+      };
+
+      config = mkOption {
+        type = types.lines;
+        default = "";
+        description = "Custom configuration for Netdata";
+      };
+
+      workDir = mkOption {
+        type = types.path;
+        default = "/var/lib/netdata";
+        description = "State directory for Netdata";
+      };
+
+      logDir = mkOption {
+        type = types.path;
+        default = "/var/log/netdata";
+        description = "Log directory for Netdata";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+
+    launchd.daemons.netdata = {
+      serviceConfig = {
+        Label = "netdata";
+        ProgramArguments = [
+          "/bin/sh" "-c"
+          "/bin/wait4path ${cfg.package}/bin/netdata &amp;&amp; exec ${cfg.package}/bin/netdata"
+        ];
+        KeepAlive = true;
+        WorkingDirectory = cfg.workDir;
+        StandardErrorPath = "${cfg.logDir}/netdata.log";
+        StandardOutPath = "${cfg.logDir}/netdata.log";
+      };
+    };
+
+    environment.etc."netdata/netdata.conf".text = cfg.config;
+
+    system.activationScripts.preActivation.text = ''
+      mkdir -p ${cfg.workDir}
+    '';
+
+  };
+}
```

```suggestion
}

```

Can you add a trailing newline at the end of the file?